hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -245,7 +246,6 @@ public void testBasicSync(boolean useJdbc, boolean useSchemaFromCommitMetadata)
     hiveClient = new HoodieHiveClient(HiveTestUtil.hiveSyncConfig, HiveTestUtil.getHiveConf(), HiveTestUtil.fileSystem);
     List<Partition> hivePartitions = hiveClient.scanTablePartitions(HiveTestUtil.hiveSyncConfig.tableName);
     List<String> writtenPartitionsSince = hiveClient.getPartitionsWrittenToSince(Option.empty());
-    writtenPartitionsSince.add(newPartition.get(0));
```

Comment:
this is actually a bug; `writtenPartitionsSince` already contains the new partition. adding this will cause 2 update events instead of 1. Not sure why it's passing all the time.

Reply:
cc @n3nash to chime in